docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,159 @@ labels:
     - cloud
     - enterprise
     - oss
-title: Tracing in Explore
+title: Traces in Explore
 weight: 20
 ---
 
-# Tracing in Explore
+# Traces in Explore
 
-You can use Explore to query and visualize traces from tracing data sources.
+You can use Explore to query and visualize traces from tracing data sources. Supported data sources include:
 
-Supported data sources are:
-
-- [Tempo]({{< relref "../datasources/tempo/" >}}) (supported ingestion formats: OpenTelemetry, Jaeger, and Zipkin)
-- [Jaeger]({{< relref "../datasources/jaeger/" >}})
-- [Zipkin]({{< relref "../datasources/zipkin/" >}})
+- [Tempo](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/)
```

Comment:
Should the URLs be fully qualified instead of starting from /docs? 
The Writers' Toolkit says to [use full URLs](https://grafana.com/docs/writers-toolkit/write/links/#link-to-grafanacom-pages) unless content is going to be shared or mounted in another location. Will this content be mounted? 

If so, you will probably need to [add the ref:](https://grafana.com/docs/writers-toolkit/write/links/#link-from-source-content-thats-reused-as-multiple-pages) for all of the links on the pages that will be mounted or shared. This could be done whenever you're ready to mount or share the pages.

Reply:
I'll have to do that once these get mounted - thank you!